### PR TITLE
GS signup fixes

### PR DESCRIPTION
**Pull request by kprosise**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

General cleanup and revisiting page.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

I *think* I like the section title changes, but if there are any objections, I do not mind keeping them as they were.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 7ead4fe1fcfddda679fa316e807d4e4f0c63cf4b are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1905/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst [hunk 1]
```diff
@@ -205,9 +205,4 @@ It can be set using 3 different methods:
 .. seealso::
    :ref:`ref-fioctl` documentation.
 
-.. _AUR Package: https://aur.archlinux.org/packages/fioctl-bin
-.. _Scoop: https://scoop.sh/
-.. _WSL: https://docs.microsoft.com/en-us/windows/wsl/install-win10
-.. _launchpad: https://launchpad.net/~fio-maintainers/+archive/ubuntu/ppa
 .. _Github Releases: https://github.com/foundriesio/fioctl/releases
-.. _Formula: https://github.com/foundriesio/homebrew-fioctl
```
Comment (vanmaegima) [inline source/getting-started/install-fioctl/index.rst:213]: Same issue from other PR, need to drop this.

File: source/index.rst [hunk 2]
```diff
@@ -66,6 +50,23 @@ OE/Yocto Project, the Linux microPlatform™ and Docker®.
...
+   reference-manual/boards/boards
+   reference-manual/qemu/qemu
+   reference-manual/factory/factory
+   reference-manual/linux/linux
+   reference-manual/ota/ota
+   reference-manual/remote-access/remote-access
+   reference-manual/security/security
+   reference-manual/testing/testing
+   reference-manual/troubleshooting/troubleshooting
+
```
Comment (vanmaegima) [inline source/index.rst:69]: Drop this as well.

**Comment by doanac** [comment]
Docs for 839eafcb099db4021601a0d155413894e6931576 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1907/docs/artifacts/html/index.html

File: source/getting-started/signup/index.rst [hunk 3]
```diff
@@ -40,28 +37,21 @@ Create one by clicking :guilabel:`Create Factory`.
...
 
-   Upon Factory creation you will be sent an email
-   with instructions to securely download your
-   :ref:`ref-offline-keys`.
-
-   It is incredibly important that your keys are kept **safe and
-   private**. Please store these keys securely.
-
-.. todo::
+   Upon Factory creation you will be sent an email with instructions to securely download your :ref:`ref-offline-keys`.
```
Comment (vanmaegima) [inline source/getting-started/signup/index.rst:40]:
```suggestion
   Upon Factory creation you will be sent an email with instructions to securely download your :ref:`ref-offline-keys`.
```
This is not the case anymore. We don't send the keys via email anymore.

Comment (vanmaegima) [reply source/getting-started/signup/index.rst:40]: Example:
Your FoundriesFactory has been created and is ready for use.

Factory Name: vanessa-check

Factory source code is located at:
  https://source.foundries.io/factories/vanessa-check/

Factory Builds: https://app.foundries.io/factories/vanessa-check/targets/

Factory Documentation: https://docs.foundries.io/latest/

If you have any questions, please send them to [support@foundries.io](mailto:support@foundries.io) or raise a ticket on Foundries.io.

Regards,
Foundries.io Team

File: source/getting-started/signup/index.rst [hunk 4]
```diff
@@ -40,28 +37,21 @@ Create one by clicking :guilabel:`Create Factory`.
...
 
-    Suggest methods of storing TuF keys securely, such as by USB in a
-    safety deposit box, or yubikey.
+   It is incredibly important that your keys are kept **safe and private**.
+   Please store these keys securely.
```
Comment (vanmaegima) [inline source/getting-started/signup/index.rst:43]:
```suggestion
   It is incredibly important that your keys are kept **safe and private**.
   Please store these keys securely.
```
We should be able to remove this as the keys are not sent by email anymore, but if we'd like to keep this for some reason, I would point to https://docs.foundries.io/latest/reference-manual/security/offline-keys.html?highlight=backup#how-to-backup-offline-keys

File: source/getting-started/signup/index.rst [hunk 5]
```diff
@@ -72,28 +62,26 @@ includes QEMU_ images for ARM_ and RISC-V_ architectures.
 .. tip::
 
    Your chosen platform determines what the initial value for the ``machines:``
-   key will be for your first build. This key and its value can later be changed
-   via ``factory-config.yml`` in the :ref:`ref-Factory-definition`
+   key will be for your first build.
+   This key and its value can later be changed via ``factory-config.yml`` in the :ref:`ref-Factory-definition`
```
Comment (vanmaegima) [inline source/getting-started/signup/index.rst:66]:
```suggestion
   This key and its value can later be changed via ``factory-config.yml`` in the :ref:`ref-Factory-definition`
```
We don't allow customers to change `machine` in their factory, they need to request support for that, same as in the section above:

> Once a Factory is created, the chosen platform/machine and Factory name cannot be changed.
> Create a new Factory or contact support if a mistake is made: https://support.foundries.io/.

Comment (kprosise) [inline source/getting-started/signup/index.rst:66]:
```suggestion
```

**Review by vanmaegima — APPROVED** [review]

File: source/getting-started/signup/index.rst [hunk 6]
```diff
@@ -40,28 +37,18 @@ Create one by clicking :guilabel:`Create Factory`.
 
 .. important::
```
Comment (kprosise) [inline source/getting-started/signup/index.rst:38]:
```suggestion
```

**Comment by doanac** [comment]
Docs for e08fa315f49fd9c4ba96a74bf92eca8859a4fe0c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1909/docs/artifacts/html/index.html

**Pull request merged**